modules/apps/portal-language/portal-language-rest-impl/src/main/java/com/liferay/portal/language/rest/internal/service/access/policy/LanguagePortalInstanceLifecycleListener.java

```diff
@@ -45,7 +45,7 @@ public void portalInstanceRegistered(Company company) throws Exception {
 	}
 
 	private void _addSAPEntries(long companyId) throws PortalException {
-		for (String[] sapEntryObjectArray : _SAP_ENTRY_OBJECT_ARRAYS) {
+		for (String[] sapEntryObjectArray : _SAP_ENTRY_ARRAYS) {
```

Comment:
Since `_SAP_ENTRY_OBJECT_ARRAYS` was renamed to `_SAP_ENTRY_ARRAYS`, the variable `sapEntryObjectArray` must reflect that

Reply:
done!